### PR TITLE
W-15059250 AEH update troubleshooting for validate rule bypass

### PR DESCRIPTION
**Pull request by JennyHajee**

### CODE REVIEW

File: modules/ROOT/pages/troubleshooting.adoc [hunk 1]
```diff
@@ -121,18 +121,65 @@ In this release, IP restrictions for the connected app are not supported. By def
...
+* *Name* 
++
+Click into the field to populate it with the label name with underscores.
++
+* *Description*
++ 
+Enter a description for the permission.
+.. Click *Save*.
+
+. Create a permission set and add the custom permission:
```
Comment (dgs87) [inline modules/ROOT/pages/troubleshooting.adoc:147]: This workaround is documented creating a new permission set, however will also possible add the custom permission to an existing permissions set like the one created for minimal permissions initiative. 
The one that is documented [here](https://docs.mulesoft.com/api-experience-hub/assigning-permissions).

Comment (JennyHajee) [reply modules/ROOT/pages/troubleshooting.adoc:147]: Thanks. I added that you can also use an existing permission set.

**Review by dgs87 — DISMISSED** [review]

File: modules/ROOT/pages/troubleshooting.adoc [hunk 2]
```diff
@@ -121,27 +121,78 @@ In this release, IP restrictions for the connected app are not supported. By def
 
 An error occurs when saving styling changes to the portal if an object has validation rules applied for standard objects that are created during the Salesforce linking process. 
 
-The workaround is to deactivate the validation rule, save the styling changes, then reactive the validation rule: 
+Validation rules cannot be applied to Salesforce standard objects, for example, account and user objects, that are created during the Salesforce linking process.
```
Comment (hannanelson) [inline modules/ROOT/pages/troubleshooting.adoc]:
```suggestion
You can't apply validation rules to Salesforce standard objects, such as account and user objects, that are created during the Salesforce linking process. 
```

File: modules/ROOT/pages/troubleshooting.adoc [hunk 3]
```diff
@@ -121,27 +121,78 @@ In this release, IP restrictions for the connected app are not supported. By def
...
+* *Name* 
++
+Click the field to populate it with the label name with underscores.
++
+* *Description*
++ 
+Enter a description for the permission.
+.. Click *Save*.
+
+. Create a permission set and add the custom permission:
```
Comment (hannanelson) [inline modules/ROOT/pages/troubleshooting.adoc:147]: It's a little weird to have the colon introducing the sub-list broken up by the hanging paragraph. Is there another place you can put the following sentence?

File: modules/ROOT/pages/troubleshooting.adoc [hunk 4]
```diff
@@ -121,27 +121,78 @@ In this release, IP restrictions for the connected app are not supported. By def
...
+AND (
+  NOT($Permission.Bypass_Account_Validation),
+  ISBLANK (Website)  
+)
+----
+.. Click *Save*.
+
+. Assign the permission set to the user performing the linking process:
++
+This user must have the System Administrator profile to perform the linking process.
```
Comment (hannanelson) [inline modules/ROOT/pages/troubleshooting.adoc]: (same as line 147)

Comment (hannanelson) [reply modules/ROOT/pages/troubleshooting.adoc]: Maybe move it to below line 184?

**Review by hannanelson — APPROVED** [review]

**Pull request merged**